### PR TITLE
made function which look for kind of number (1, -1, 0)

### PR DESCRIPTION
**Pull request by O-Julia-O**
Created a function which looks for type of number

### CODE REVIEW

File: showSignNumbers.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+/* Используя конструкцию if..else, напишите код, который получает число через prompt, а затем выводит в alert:
+
+1, если значение больше нуля,
+-1, если значение меньше нуля,
+0, если значение равно нулю.
+Предполагается, что пользователь вводит только числа. */
+
+let questionAboutNum = +prompt("Введите число: ", "")
```
Comment (Kikubari) [inline showSignNumbers.js:8]: why + is here?

File: showSignNumbers.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    alert("0")
+}
+else if (questionAboutNum > 0) 
+{
+    alert("1")
+}
+else if (questionAboutNum < 0)
+{
+    alert("-1")
+}
```
Comment (Kikubari) [inline showSignNumbers.js:21]: a ternary condition operator would be better option

**Review by Kikubari — APPROVED** [review]

File: showSignNumbers.js (hunk 1 above)
Comment (O-Julia-O) [reply showSignNumbers.js:8]: It is changed type of values. Here I get a string and exchange type of number

File: showSignNumbers.js (hunk 2 above)
Comment (O-Julia-O) [reply showSignNumbers.js:21]: yep, it would be better but in this case I had to create this with if.

**Pull request merged**